notification-service/src/test/java/com/redhat/parodos/notification/validation/ValidNotificationMessageUserOrGroupValidatorTest.java

```diff
@@ -0,0 +1,63 @@
+package com.redhat.parodos.notification.validation;
+
+import java.util.List;
+
+import javax.validation.ConstraintValidatorContext;
+
+import com.redhat.parodos.notification.dto.NotificationMessageCreateRequestDTO;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+public class ValidNotificationMessageUserOrGroupValidatorTest {
+
+	private ValidNotificationMessageUserOrGroupValidator validator;
+
+	private ConstraintValidatorContext constraintValidatorContext;
+
+	@Before
+	public void setUp() {
+		validator = new ValidNotificationMessageUserOrGroupValidator();
+		constraintValidatorContext = mock(ConstraintValidatorContext.class);
+	}
+
+	@Test
+	public void shouldReturnTrueWhenValidUsernames() {
+		NotificationMessageCreateRequestDTO notificationMessage = new NotificationMessageCreateRequestDTO();
+		notificationMessage.setUsernames(List.of("test"));
+
+		assertThat(validator.isValid(notificationMessage, constraintValidatorContext)).isTrue();
```

Comment:
please, replace all `assertThat(....).isTrue()` with `assertTrue`

Reply:
As discussed with @masayag this is the convention used in spring projects and going to be enforce in this project